File: 1-easy/problems/0961-n-repeated-element-in-size-2n-array/solution.ts
function repeatedNTimes(A: any): boolean | number | string | any {
  """
  :type A: List[int]
  :rtype: int
  """
  for (i in Array.from({length: 2, (A}, (_, i) => i).length)) {
      if (A[i-1] == A[i] || A[i-2] == A[i]) {
          return A[i];
  return A[0];
}

export { repeatedNTimes };

File: 1-easy/problems/0961-n-repeated-element-in-size-2n-array/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def repeatedNTimes(self, A):
        """
        :type A: List[int]
        :rtype: int
        """
        for i in range(2, len(A)):
            if A[i-1] == A[i] or A[i-2] == A[i]:
                return A[i]
        return A[0]
